library/core/registry/src/main/java/org/quiltmc/qsl/registry/mixin/SimpleRegistryMixin.java

```diff
@@ -51,8 +90,121 @@ protected SimpleRegistryMixin(RegistryKey<? extends Registry<V>> key, Lifecycle
 			at = @At("RETURN")
 	)
 	private void quilt$invokeEntryAddEvent(int rawId, RegistryKey<V> key, V entry, Lifecycle lifecycle, boolean checkDuplicateKeys,
-	                                       CallbackInfoReturnable<Holder<V>> cir) {
+										   CallbackInfoReturnable<Holder<V>> cir) {
```

Comment:
Extremely nit-picky but I would keep the spaces from smart-tabs as it keeps alignment.